public/app/features/alerting/unified/components/rule-editor/util.test.ts

```diff
@@ -25,7 +26,7 @@ describe('rule-editor', () => {
       type: 'classic_conditions',
       datasource: {
```

Comment:
Even better... We could replace the whole datasource object with the constant ref

Reply:
good call. Pushed